frontend/projects/abtesting/src/app/core/users/store/users.effects.ts

```diff
@@ -106,6 +106,18 @@ export class UsersEffects {
     )
   );
 
+  deleteUser$ = createEffect(() =>
+    this.actions$.pipe(
+      ofType(UsersActions.actionDeleteUser),
+      map(action => action.email),
+      filter(( email ) => !!email ),
```

Comment:
@YashilDepani  Can you please fix formatting here?

Reply:
code updated